.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["plugin:@typescript-eslint/recommended", "prettier"]
```

Comment:
Can we move this file contents to packege.json? Similar to other configurations.

Reply:
I think the editor plugin for eslint picks up the config easily this way.what do you think @simison ?

Reply:
Editors pick up whatever config. Either in package.json, or in .eslintrc.

Reply:
Ok.